src/strategies/seedify-cumulative-voting-power-hodl-staking-farming/index.ts

```diff
@@ -20,13 +21,20 @@ export async function strategy(
   options,
   snapshot
 ): Promise<Record<string, number>> {
+  if (options.sfundStakingAddresses.length > 10) {
```

Comment:
Actually, we only restricted by multicall and we can find maximum allowed parameters. However, if there are no plan to use more than 10 addresses, it doesn't matter to find maximum

Reply:
indeed, though multicall does not have a limit to my understanding and what snapshot.org told me

I'm adding this limit as we never use more than 10 addresses and also allows snapshot.org to get a clear amount of addresses we will use